cmd/bosun/expr/funcs.go

```diff
@@ -104,24 +107,35 @@ var TSDB = map[string]parse.Func{
 		parse.TypeSeries,
 		tagQuery,
 		Band,
+		nil,
 	},
 	"change": {
 		[]parse.FuncType{parse.TypeString, parse.TypeString, parse.TypeString},
 		parse.TypeNumber,
 		tagQuery,
 		Change,
+		nil,
 	},
 	"count": {
 		[]parse.FuncType{parse.TypeString, parse.TypeString, parse.TypeString},
 		parse.TypeScalar,
 		nil,
 		Count,
+		nil,
```

Comment:
I looked into merging it with the Tags function, but it wasn't a good fit. However I can switch to named parameters, which will eliminate the need for nil if unused.